pkg/config/config.go

```diff
@@ -7,27 +7,35 @@ import (
 var (
 	accessTokenField = field.StringField(
 		"token",
+		field.WithDisplayName("Personal access token"),
 		field.WithDescription("The GitHub access token used to connect to the GitHub API."),
+		field.WithRequired(true),
```

Comment:
I think we have to remove required here because of how we validate in baton-sdk, Ben removed this recently to support auth with a github app. Let's remove `WithRequired` on this for now, and leave a TODO to look into this

Reply:
@laurenleach Oh, good catch! I actually meant to ask about this.